### ISSUE TITLE
Enable tab navigation through speakers

### ISSUE THREAD

**Issue by alexnm** [cluj-javascripters/jsheroes-2018#170]
- [x] speaker boxes are not focusable by keyboard navigation
- [x] links should also be links, not divs
- [x] we should test with a screen reader to ensure proper usage
